README.md

```diff
@@ -27,9 +27,17 @@ Once a bot is registered, you can invite to a channel with `/invite @slackbotser
 
 ### Run Your Own
 
-You can use the [sample application](sample_app) to bootstrap your project and start adding slack command handlers on top of this code.
+You can use the [sample application](sample_app) to bootstrap your project and start adding slack command handlers on top of this code. A database is required to store teams.
 
-Install [MongoDB](https://www.mongodb.org/downloads), required to store teams. We would like your help with [supporting other databases](https://github.com/slack-ruby/slack-ruby-bot-server/issues/12).
+### MongoDB
+
+Use [Mongoid](https://github.com/mongodb/mongoid) as ODM. Configure the database connection in `mongoid.yml`. See the [MongoDB example](sample_apps/sample_app_mongoid) for more information.
+
+### PostgreSQL
+
+Use the [pg](https://github.com/ged/ruby-pg) interface. Configure the database connection in `postgresql.yml`. See the [PostgreSQL example](sample_apps/sample_app_pg) for more information.
```

Comment:
Yeah! I mean that actually opens things up a lot, kind of interesting. ActiveRecord has adapters for SQlite3 and several other DBMSs, I think.